.golangci.yml

```diff
@@ -12,26 +12,41 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+version: "2"
+run:
+  issues-exit-code: 1
 linters:
   enable:
-  - errcheck
-  - gofmt
-  - goimports
-  - gosec
-  - gocritic
-  - misspell
-  - revive
-  - unused
-output:
-  uniq-by-line: false
-issues:
-  exclude-rules:
-  - path: _test\.go
-    linters:
-    - errcheck
+    - gocritic
     - gosec
+    - misspell
+    - revive
+  exclusions:
+    generated: lax
+    presets:
+      - comments
+      - common-false-positives
+      - legacy
+      - std-error-handling
+    rules:
+      - linters:
+          - errcheck
+          - gosec
+        path: _test\.go
+    paths:
+      - third_party$
+      - builtin$
+      - examples$
+issues:
   max-issues-per-linter: 0
   max-same-issues: 0
-run:
-  issues-exit-code: 1
-  timeout: 10m
\ No newline at end of file
+formatters:
+  enable:
+    - gofmt
```

Comment:
oh cool, can these be applied? or just checked?

Reply:
I have been manually applying the results currently.